src/renderer/components/dialogs/CreateChat.tsx

```diff
@@ -367,6 +379,35 @@ export function AddMemberInnerDialog({
   useLayoutEffect(applyCSSHacks, [inputRef, contactIdsToAdd])
   useEffect(applyCSSHacks, [])
 
+  const addContactOnClick = async () => {
+    if (!queryStrIsValidEmail) return
+
+    await DeltaBackend.call('contacts.createContact', queryStr)
+
+    if (inputRef.current) {
+      const nativeInputValueSetter = Object.getOwnPropertyDescriptor(
+        window.HTMLInputElement.prototype,
+        'value'
+      )?.set
+      nativeInputValueSetter?.call(inputRef.current, queryStr.split('@')[0])
+      var inputEvent = new Event('input', { bubbles: true })
+      inputRef.current.dispatchEvent(inputEvent)
```

Comment:
I assume this code is for updating the search results after creating the new contact, right?

Reply:
Aaaah this writes the email into the input field. I think you could do `updateContacts()` to refetch contacts and `onSearchChange(inputEvent)` to change the search input. This is probably a bit more error prone then doing it through the dom.

Reply:
Hmm I think a better approach would be to call `updateContacts()` & `onSearchChange(inputEvent)`

Reply:
I know! I was trying to use the onChange handler but it is only called when the input changes directly in the application. This is the way to trigger the onChange handler "manually" or programatically.

Reply:
If it is not updated then after adding a new contact you have to manually change the input again to be able to find the contact